daemon/cmd/daemon_main.go

```diff
@@ -1835,7 +1831,7 @@ func startDaemon(d *Daemon, restoredEndpoints *endpointRestoreState, cleaner *da
 
 	bootstrapStats.healthCheck.Start()
 	if option.Config.EnableHealthChecking {
-		d.initHealth(params.HealthAPISpec, cleaner)
+		d.initHealth(params.HealthAPISpec, cleaner, params.Sysctl)
```

Comment:
non-blocking nit: Commit message still mentions "IOW where a statedb and the
reconcilers are not available (e.g: cilium-health and cilium-cni)", even though cilium-health does use the reconciler

Reply:
Right :face_exhaling: